### PR TITLE
Homework 7

### PR DESCRIPTION
**Pull request by AbdurakhimovAziz**
As I did the most part of hw7 in previous pr, I don't have so many additions.

### CODE REVIEW

File: client/src/app/accounts/accounts-item/accounts-item.component.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-accounts-item',
+  templateUrl: './accounts-item.component.html',
+  styleUrls: ['./accounts-item.component.scss'],
+})
+export class AccountsItemComponent {
+  public title: string = 'Debit Card';
```
Comment (AnastasiaHarost) [inline client/src/app/accounts/accounts-item/accounts-item.component.ts:9]: looks like mock data

File: client/src/app/accounts/accounts-list/accounts-list.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<app-accounts-item></app-accounts-item>
```
Comment (AnastasiaHarost) [inline client/src/app/accounts/accounts-list/accounts-list.component.html:1]: use *ngFor

File: client/src/app/app.module.ts [hunk 3]
```diff
@@ -23,6 +28,10 @@ import { HttpClientModule } from '@angular/common/http';
     NotFoundComponent,
```
Comment (AnastasiaHarost) [inline client/src/app/app.module.ts:28]: let's move all components, from app.module, in declarations should be only app.component. In imports you can add module with all this components

File: client/src/app/accounts/accounts-item/accounts-item.component.ts (hunk 1 above)
Comment (AbdurakhimovAziz) [reply client/src/app/accounts/accounts-item/accounts-item.component.ts:9]: yes, I will change it as soon as I update backend

File: client/src/app/modules/components.module.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import { LoginComponent } from '../login/login.component';
+import { MainComponent } from '../main/main.component';
+import { NotFoundComponent } from '../not-found/not-found.component';
+import { StatisticsComponent } from '../statistics/statistics.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { AppRoutingModule } from '../app-routing.module';
+
+@NgModule({
+  declarations: [
+    HeaderComponent,
```
Comment (AnastasiaHarost) [inline client/src/app/modules/components.module.ts:19]: inside client/src/app you should have AppComponent, AppModule, shared folder and some new folder, for example, components. Inside this components folder should be all components and new module

Comment (AnastasiaHarost) [reply client/src/app/modules/components.module.ts:19]: As I see ComponentsModule and app-material.module is the same, remove one. You should have 2 modules

Comment (AbdurakhimovAziz) [reply client/src/app/modules/components.module.ts]: > As I see ComponentsModule and app-material.module is the same, remove one. You should have 2 modules

Actually, they are different 
`app-material.module.ts`
image

File: client/src/app/app-routing.module.ts [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
-import { MainComponent } from './main/main.component';
-import { NotFoundComponent } from './not-found/not-found.component';
-import { StatisticsComponent } from './statistics/statistics.component';
+import { CategoriesComponent } from './components/categories/categories.component';
+import { LoginComponent } from './components/login/login.component';
+import { MainComponent } from './components/main/main.component';
+import { NotFoundComponent } from './components/not-found/not-found.component';
+import { StatisticsComponent } from './components/statistics/statistics.component';
 
 const routes: Routes = [
```
Comment (AnastasiaHarost) [inline client/src/app/app-routing.module.ts:10]: and this routes should be in components-roting.module.ts in folder components

Comment (AnastasiaHarost) [reply client/src/app/app-routing.module.ts:10]: in this case you don't need to export all component

**Review by AnastasiaHarost — COMMENTED** [review]
main folder is empty, you can remove it

**Comment by AbdurakhimovAziz** [comment]
> main folder is empty, you can remove it

It is not empty, maybe github is displaying the folder, which is moved into components folder
Here is my folder structure
image

**Pull request merged**